docs/content/docs/2.components/chat-messages.md

```
@@ -439,6 +439,14 @@ You can use all the slots of the [`ChatMessage`](/docs/components/chat-message#s
 ```
 ::
 
+### Expose
+
+When accessing the component via a template ref, you can use the following:
+
+| Name | Type |
+| ---- | ---- |
+| `registerMessageRef(id: string, element: ComponentPublicInstance | null)`{lang="ts-type"} | `void`{lang="ts-type"} |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)